### ISSUE TITLE
Running App: Error in open.connection(con, "rb") : HTTP error 401.

### ISSUE THREAD

**Issue by mjones01** [Danielslee51/Calliope-View#2]
@Danielslee51 Looks like a really interesting app. 

I'm trying to run your app but I keep running into the error: `Error in open.connection(con, "rb") : HTTP error 401` when executing the `server.R` script.  

Looking into it some others have encountered similar things when pulling lots of data in other use cases: (for example: https://github.com/ropensci/GSODR/issues/10 ).  

Without adding print messages throughout the code, I haven't figured out exactly where it is stopping but I am getting both the FieldSite_point and FieldSite_point_JSON data, each has 81 observations for all the field sites.  Any suggestions would be appreciated.

**Comment by Danielslee51** [comment]
Thank you for alerting me, I know what is going on. Basically some of the NEON data comes in static files from [here](https://www.neonscience.org/data/maps-spatial-data), but other data such as drone data and the field site boundaries, whose code is located right under the code for FieldSite_point and FieldSite_point_JSON, is pulled from indexes that are created by some of our guys at [CyVerse](http://www.cyverse.org/) (`FieldSite_poly_JSON <- fromJSON('http://128.196.38.73:9200/neon_sites/_search?pretty')`). Just yesterday David (who manages the indexes) took them down and added authentication for them, so they way they work changed, and I've only just now gotten the chance to address it (I'm on vacation preparing for a climb). Since they changed, the line above is no longer valid and returns the error you saw. However, before he took them down I saved all the contents of the indexes into the local files and I added them to Calliope View to use until the drone indexes are reintegrated into the app (which will be a bit since I'm prioritizing NEON right now). I just haven't gotten a change to commit those changes, and I'm planning on doing that soon (today). After that everything will work, sorry for the inconvenience!
